Code-4-Community/speak-for-the-trees-frontend
# reservation modal attempt

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

--
-handleOk and handleCancel have been moved so they can be added to containers
-set visibility also moved so can be added to containers

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

--
Implemented requested changes, just needs final look over hopefully.

## Screenshots

Provide screenshots of any new components, styling changes, or pages.

--
block reserved by user:
<img width="520" alt="Screen Shot 2021-02-07 at 3 02 26 PM" src="https://user-images.githubusercontent.com/40365932/107162088-b4b79980-6955-11eb-9031-26a9f8963aed.png">
block reserved by another user:
<img width="518" alt="Screen Shot 2021-02-07 at 3 04 21 PM" src="https://user-images.githubusercontent.com/40365932/107162105-de70c080-6955-11eb-8da2-694fe308ef95.png">
block available to reserve:
<img width="518" alt="Screen Shot 2021-02-07 at 3 03 44 PM" src="https://user-images.githubusercontent.com/40365932/107162096-c8630000-6955-11eb-84c8-84af309bbdba.png">



## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

--
To verify my changes work, I imported my modal in the login file and made a mock modal in the login page and would change its fields to mimic the different scenarios of blocks to ensure the modal that popup is as expected.
